Vagrant/bibdataVagrantfile

```diff
@@ -0,0 +1,58 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  # Vagrant configuration options are fully documented at
+  # https://docs.vagrantup.com.
+
+  config.vm.box = "princeton_box"
+  config.vm.network "private_network", type: "dhcp"
+  config.vm.network "forwarded_port", guest: 80, host: 8484 # apache web server in production mode
+  config.vm.network "forwarded_port", guest: 8983, host: 9893  # solr
```

Comment:
Do we need this? I didn't think bibdata used Solr...

Reply:
Nope. This is extraneous.